database/seeders/EasyUserSeeder.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\User;
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\Hash;
+
+class EasyUserSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $admin = User::where('email', 'admin@email.com')->first();
+        if ($admin){
+               $admin->name = 'Joseph doe';
+               $admin->email = 'admin@email.com';
+               $admin->password = Hash::make('password');
+               $admin->save();
+        }else{
+            User::create([
+                'name' => 'Joseph doe',
+                'email' => 'admin@email.com',
+                'password' => Hash::make('password'),
+            ]);
+        }
```

Comment:
We should replace this with the user factory calls

Reply:
why? the whole point of this is to create a user we can login with. It makes a default. A factory makes a random user, with a random password. This at least allows for a generic account